include/tester/vtcp/VTCPSession.hpp

```diff
@@ -18,8 +18,11 @@ class VTCPSession : public TCPSession
     std::function<void()> startManualTest,
     std::function<bsoncxx::document::value()> getHistoryHeaders,
     std::function<bsoncxx::document::value(const std::string &)> getHistoryLog):
-        TCPSession(std::move(socket), id, onMessageReceived, onDisconnect, startAutoTest, 
-            startManualTest, getHistoryHeaders, getHistoryLog)
+        TCPSession(std::move(socket), id, onMessageReceived, onDisconnect),
+        _startAutoTest(startAutoTest),
```

Comment:
same for these 4 members - use `std::move` on the functions you pass by value.

Reply:
done